sql-db/src/test/java/io/thorntail/openshift/ts/sql/db/ExternalOracleIT.java

```diff
@@ -6,6 +6,6 @@
 @RunWith(Arquillian.class)
 public class ExternalOracleIT extends AbstractExternalSqlDatabaseTest {
     public ExternalOracleIT() {
-        super("oracle12cR2&&geo_BOS", "target/test-classes/project-defaults-external-oracle.yml");
+        super("oracle12c", "target/test-classes/project-defaults-external-oracle.yml");
```

Comment:
No, we need `oracle12cR2`. Not sure if `geo_BOS` makes a difference...

Reply:
It seems we have `oracle12cR2` in both geos.

Reply:
I think I already tested with oracle12cR2 on Friday. I am retesting now.